docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,306 @@
+- `"recognition_model"`: The model to use for facial recognition. `"ArcFace"` is chosen as the default for a good balance of speed and accuracy. See [GitHub](https://github.com/viam-labs/facial-detection) for more options.
+- `"detection_framework"`: The detection framework to use for facial detection. `"ssd"` is chosen as the default for a good balance of speed and accuracy.
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "face_labels": {
+    "my_name": "/home/me/my-photo.jpg"
+  },
+  "recognition_model": "ArcFace",
+  "detection_framework": "ssd"
+}
+```
+
+Make sure to name this detector `"face-detect"`.
+
+## Configure a verification system
+
+Now that you've configured your camera, a rough `"people-detect"` detector and a more fine-tuned `"face-detect"` facial detector, you can add a module that contains the code that runs the alarm system based on detections it receives from the detectors specified in its configuration.
+
+Navigate to the **Config** tab of your Machine’s page in [the Viam app](https://app.viam.com/). Click on the **Services** subtab and click **Create service**. Select the `vision` type, then select the `classifier:verification-system` model. Enter a name for your vision service and click **Create**.
+
+To use the facial detector as the verification detector and the people detector as the first two layers of detectors, with **Builder** mode selected copy and paste the following attributes configuration into your service's **Attributes** box:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "trigger_1_confidence": 0.35,
+  "verification_detector": "face-detect",
+  "camera_name": "my-webcam",
+  "trigger_2_confidence": 0.5,
+  "trigger_1_labels": ["Person"],
+  "trigger_2_labels": ["Person"],
+  "disable_alarm": false,
+  "trigger_2_detector": "people-detect",
+  "verification_labels": ["my_name"],
+  "trigger_1_detector": "people-detect",
+  "disarmed_time_s": 10,
+  "countdown_time_s": 10
+}
+```
+
+Name your service `"security"`.
+
+- Trigger detectors 1 and 2 cascade to the alarm state.
+  Since the `"people-detect"` detector serves as both trigger detector 1 and 2, set `"trigger_1_labels"` and `"trigger_2_labels"` to `"Person"` to match the bounding box label of `"Person"` you assigned to images in your dataset when training the object detection model behind this detector.
+- As the identity verification detector is your facial detector `"face-detect"`, set `"verification_labels"` to match each name you assigned to an image in the facial detector modules' `"face_labels"` configuration attribute.
+- Note that `"camera_name": "my_webcam"` matches the name we configured for our source camera.
+  If you used a different name for your camera, change the name in the JSON.
+- Edit the other attributes to reflect your desired confidence thresholds and times between states.
```

Comment:
Nit: Let's render this as steps also:

````suggestion
1. On your machine's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Services** tab.
1. Click the **Create services** button at the bottom of the page, select `vision`, then select the `[filtered-camera](classifier:verification-system)` model.
   You can also search for `verification-system` directly.
1. Name your modular vision service `security`, then click **Create**.
1. On the panel that appears, enter the following configuration into the **Attributes** field:

   ```json {class="line-numbers linkable-line-numbers"}
   {
     "trigger_1_confidence": 0.35,
     "verification_detector": "face-detect",
     "camera_name": "my-webcam",
     "trigger_2_confidence": 0.5,
     "trigger_1_labels": ["Person"],
     "trigger_2_labels": ["Person"],
     "disable_alarm": false,
     "trigger_2_detector": "people-detect",
     "verification_labels": ["my_name"],
     "trigger_1_detector": "people-detect",
     "disarmed_time_s": 10,
     "countdown_time_s": 10
   }
   ```

   In the configuration above:

   - `"trigger_1_detector"` and `"trigger_2_detector"` both use the `people-detect` ML model you created to determine if a person is present in the camera frame.
      For this tutorial, you are configuring both of these triggers identically to use the person detection ML model. 
   - `"trigger_1_labels"` and `"trigger_2_labels"` similarly both use the `"person"` label you added to images when training the `people-detect` model.
      For this tutorial, you are configuring both of these labels identically to use the person detection ML model. 
   - `"verification_detector"` uses the `face-detect` ML model you configured when you added images of faces to approved and labelled them in the configuration.
   - `"verification_labels"` contains an array of approved names that match each name you assigned to an image in the `facial-detector` modules' `"face_labels"` configuration attribute.
   - `"camera_name"` is the name of the camera to use to detect people and faces.
      If you used a different name for your camera, update this parameter with your camera's name.
   - Edit the other attributes to reflect your desired confidence thresholds and times between states.

See the [`verification-system` module documentation](https://github.com/viam-labs/verification-system) for more information about the trigger states and their various configuration options.
````

Anticipate: users will intuit trigger_1 means person-detect and trigger_2 means face-detect, but this is not so. So I think we definitely need to tell them that, for the purposes of this tutorial, we're not using a more advanced feature of the module whereby we could use 1 and 2 separately. I don't know what that could be, but I'm inferring from this copy and the code that we're not using it here, and that trigger_1 and trigger_2 should be configured identically in this tutorial's conf. I have offered a guess in my above suggestion, but please correct as you know!!